src/nunit-gui/Model/TestModel.cs

```diff
@@ -240,8 +240,7 @@ public void UnloadTests()
             _files = null;
             _resultIndex.Clear();
 
-            if (TestUnloaded != null)
-                TestUnloaded(new TestEventArgs(TestAction.TestUnloaded));
```

Comment:
Why the removal of the guard? (I guess that right now we can be sure that there are handlers associated with the event, but that might not be case in the future?)